lib/score.rb

```diff
@@ -0,0 +1,110 @@
+module Scrabble
+  class Score
+    attr_reader :words
+    def initialize
+      @words = []
+    end
+
+    # calculates the score of a given word
+    def self.score(word)
+	  word = word.upcase
+      score_chart =
+      {
+        1 => ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+        2 => ["D", "G"],
+        3 => ["B", "C", "M", "P"],
```

Comment:
This data structure certainly works, but it means you need a double loop below. It might be a little cleaner to have the map go the other way, that is, from letters to score. Then finding a letter's score is as concise as `score_chart[letter]`.